### DISCUSSION TITLE
Ring of "desolation"

### DISCUSSION THREAD

**Discussion by CaptianRage** [Aizistral-Studios/Enigmatic-Legacy#450] [Q&A]
How to remove the desolation_ring ?
![image](https://github.com/Aizistral-Studios/Enigmatic-Legacy/assets/149879204/6830ba5d-7a2f-475b-98ff-831d48f88537)

**Comment by SoulRacer65** [comment]
not possible without the use of creative or commands, it even states in the tooltip that it's not unequipable
